File: app/src/main/java/dev/ragnarok/fenrir/picasso/Content_Local.java
package dev.ragnarok.fenrir.picasso;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@IntDef({Content_Local.PHOTO,
        Content_Local.VIDEO,
        Content_Local.AUDIO})
@Retention(RetentionPolicy.SOURCE)
public @interface Content_Local {
    int PHOTO = 1;
    int VIDEO = 2;
    int AUDIO = 3;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/picasso/Content_Local.kt
package dev.ragnarok.fenrir.picasso

import androidx.annotation.IntDef

@IntDef(Content_Local.PHOTO, Content_Local.VIDEO, Content_Local.AUDIO)
@Retention(AnnotationRetention.SOURCE)
annotation class Content_Local {
    companion object {
        const val PHOTO = 1
        const val VIDEO = 2
        const val AUDIO = 3
    }
}